pokemongo_bot/cell_workers/catch_visible_pokemon.py

```diff
@@ -9,17 +8,26 @@
 class CatchVisiblePokemon(BaseTask):
     def work(self):
         if 'catchable_pokemons' in self.bot.cell and len(self.bot.cell['catchable_pokemons']) > 0:
-            logger.log('Something rustles nearby!')
             # Sort all by distance from current pos- eventually this should
             # build graph & A* it
             self.bot.cell['catchable_pokemons'].sort(
                 key=
-                lambda x: distance(self.bot.position[0], self.bot.position[1], x['latitude'], x['longitude']))
+                lambda x: distance(self.bot.position[0], self.bot.position[1], x['latitude'], x['longitude'])
+            )
 
-            user_web_catchable = 'web/catchable-%s.json' % self.bot.config.username
             for pokemon in self.bot.cell['catchable_pokemons']:
-                with open(user_web_catchable, 'w') as outfile:
```

Comment:
Removing this means that the web uis will be completely unusable until they support the events. I think we should leave these lines in to support the old UIs until we have a new UI that works with the event system.